src/core/server/logging/logger.test.ts

```diff
@@ -27,7 +27,7 @@ let appenderMocks: Appender[];
 let logger: BaseLogger;
 
 const timestamp = new Date(2012, 1, 1);
-jest.spyOn(global, 'Date').mockImplementation(() => timestamp);
+jest.spyOn<any, any>(global, 'Date').mockImplementation(() => timestamp);
```

Comment:
This spy doesn't ever seem to be restored with `mockRestore`. We should make sure that happens explicitly after these tests run since we don't have the `restoreMocks` config option set to true.

As an aside: it'd be awesome if there was linter rule we could add for this.

Reply:
👍 will fix. don't know why but I was sure we use `restoreMocks: true`